### DISCUSSION TITLE
Restart stuck at first timestep

### DISCUSSION THREAD

**Discussion by fcurow** [firemodels/fds#13595] [General]
To whom it may concern,

Have you guys experienced restarting an fds run and it just gets stuck on the first timestep? We have waited a day for it to progress (restarted the case from time zero in parallel).

Best Regards,
Sal

**Comment by rmcdermo** [comment]
I am a bit confused.  If you are starting from T=0, do you need the restart?

Please attach you input file and we will test it.  Thanks

**Reply by fcurow** [reply]
If I restart from restart files, i.e. ~200 sec, it hangs at that time and does not progress. The fds file works fine if I start from 0 sec.

**Reply by rmcdermo** [reply]
OK, please send us the file.

**Reply by fcurow** [reply]
It's project sensitive, I can't share it.  Any specific lines I should look out for?

**Reply by rmcdermo** [reply]
What version of FDS are you using?  Since we started adding RESTART verification cases some time back, I have noticed a significant reduction in RESTART problems.  Please make sure you are using the latest code.  Beyond this, we would need some kind of test to debug.  Try to reproduce the problem in some non-sensitive case.

**Reply by fcurow** [reply]
version 6.7.5

**Reply by rmcdermo** [reply]
Go the [FDS Release Notes](https://github.com/firemodels/fds/wiki/FDS-Release-Notes) and search on "restart".  You will see many issues addressed between 6.7.5 (2020) and now.  Please try with the latest code and let us know.  Thanks
